systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -654,6 +673,71 @@ private void deleteClusterOperator() {
         LOGGER.info("Cluster Operator Pod deleted");
     }
 
+    /**
+     * Based on {@param withJbodStorage} it returns the {@link Storage} - either with just one Persistent Volume, or with
+     * JBOD storage containing two Persistent Volumes.
+     *
+     * @param withJbodStorage   boolean parameter determining if the JBOD storage should be used in the test case
+     *
+     * @return  {@link Storage} for the particular testcase (single PV/JBOD)
+     */
+    private Storage getStorageBasedOnTestCase(boolean withJbodStorage) {
+        String pvSize = "1Gi";
+
+        PersistentClaimStorage volume = new PersistentClaimStorageBuilder()
+            .withDeleteClaim(true)
+            .withId(METADATA_VOLUME_ID)
+            .withKraftMetadata(KRaftMetadataStorage.SHARED)
+            .withSize(pvSize)
+            .build();
+
+        if (withJbodStorage) {
+            PersistentClaimStorage secondVolume = new PersistentClaimStorageBuilder()
+                .withDeleteClaim(true)
+                .withId(1)
+                .withSize(pvSize)
+                .build();
+
+            return new JbodStorageBuilder()
+                .withVolumes(volume, secondVolume)
+                .build();
+        }
+
+        return volume;
+    }
+
+    /**
+     * Method returning simple stream of arguments for migration parametrized tests.
+     * First argument determines if the ZK delete claim should be set to true/false, second if
+     * the CO should be deleted during the migration/rollback process, and third if the JBOD storage should be used.
```

Comment:
Could we write this as 
```
 * <ul>
 *     <li>$1 - Deletes Zookeeper claim: true, false</li>
 *     <li>$2 - Deletes Cluster Operator during the process: false, true</li>
 *     <li>$3 - Uses JBOD storage configuration: true, false</li>
 * </ul>
```
or something similar? :)

Reply:
sure, do you want from me to change it everywhere in the MigrationST to ordered lists in HTML tag? I mean the steps etc.

Reply:
No, I think just part:
```
 * First argument determines if the ZK delete claim should be set to true/false
 ```
 to 
 ```
 $1 - Deletes Zookeeper claim: true, false
 ...
 etc.
 ```
 Would be sufficient enough.

Reply:
ok, will do